benchmarks/cpp/tensorexpr/bench_reduce.cpp

```diff
@@ -266,10 +266,9 @@ BENCHMARK_DEFINE_F(Reduce1D, TeSplitTail)(benchmark::State& state) {
   {
     auto const& loops = loop.getLoopStmtsFor(BT);
     te::For* m = loops[1];
-    te::For* mo;
     te::For* mi;
     te::For* tail;
-    loop.splitWithTail(m, kChunkSize, &mo, &mi, &tail);
+    loop.splitWithTail(m, kChunkSize, &mi, &tail);
```

Comment:
Nit: can remove `mi` and `tail` entirely since they are unused (here and in other places too).

Reply:
Done.